examples/menubar/menubar-1/menubar-1.html

```diff
@@ -149,7 +149,9 @@ <h3 id="ex1_label">Example 1: Menubar With ARIA Markup in the HTML Source</h3>
           to easily see which elements the roles and properties need to be included.</li>
         <li>This is useful for people writing their own scripts to see which elements need
           roles and properties.</li>
-      </ol>
+        <li>CSS <code>:after</code> is used to render the down arrow in a menubar item using the CSS <code>content</code> property string using entity values.</li>
```

Comment:
Using content property causes screen readers to announce the arrow characters ... we do not want that.